### DISCUSSION TITLE
unique selector

### DISCUSSION THREAD

**Discussion by Art-Che** [AutomaApp/automa#1061] [Q&A] [answered]
Hi!
I need to fill in the field In CRM. Problem, some items have a unique ID:

.side-panel-iframe |> .popup-window:nth-child(**391**) .popup-multiselect-content > div

(391) the value changes every time.

my selector settings
![image](https://user-images.githubusercontent.com/120256667/210918293-6171073d-607f-4395-87c9-6e7f868c5845.png)

What can be done?

**Comment by Kholid060** [comment, marked as answer]
Have you tried using XPath? If it's not working, you must write the selector manually. If you don't know how you can follow these tutorials

https://vegibit.com/css-selectors-tutorial/
https://www.guru99.com/locators-in-selenium-ide.html

**Reply by Art-Che** [reply]
Thank you for your prompt response. I'll try.
